style_editor.html

```diff
@@ -26,6 +25,15 @@
   vertical-align: top;
   width: 50%;
 }
+/* ProffieOS Style Editor */
```

Comment:
A nice page title for the top of the page.

Reply:
Seems like a waste of space to me.

Reply:
aww. Thought some branding would be good? Nothing going on up there anyway. Of course it's your call. Maybe it'll grow on ya? Maybe smaller? Add logo links to The Crucible ,POD, GitHub, homepage?

Reply:
The problem is that any space that we use on top ends up eating up space on the bottom, which tends to be fairly sparse on laptops and other devices with a screen that is 16:9 or wider.

Reply:
Understood. How about a footer instead way down out of the way?
I'll make it it's own PR.